apps/server/src/modules/task/service/submission.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import { Counted, EntityId, Submission } from '@shared/domain';
+import { SubmissionRepo } from '@shared/repo';
+
+@Injectable()
+export class SubmissionService {
+	constructor(private readonly submissionRepo: SubmissionRepo) {}
+
+	async findAllByTask(taskId: EntityId): Promise<Counted<Submission[]>> {
```

Comment:
Do we have a rule now that we always want to have a service layer between UC and repo? In a lot of UCs there are repos injected and directly used. I'm asking because the service method here is pure forwarding.

Reply:
Ja haben wir als regeln, UC sollen keine Repos direkt verwenden